pkg/controller/migmigration/quiesce.go

```diff
@@ -580,30 +580,65 @@ func (t *Task) unQuiesceDaemonSets(client k8sclient.Client, namespaces []string)
 // Suspends all CronJobs
 func (t *Task) quiesceCronJobs(client k8sclient.Client) error {
 	for _, ns := range t.sourceNamespaces() {
-		list := batchv1beta.CronJobList{}
-		options := k8sclient.InNamespace(ns)
-		err := client.List(context.TODO(), &list, options)
+		cClient, err := t.getSourceClient()
```

Comment:
Instead of grabbing src client again here, can we just change the arg type for `quiesceCronJobs` from `k8sclient.Client` to `compat.Client`?